frontend/webapp/containers/main/destinations/add-destination/index.tsx

```diff
@@ -26,31 +24,57 @@ const NotificationNoteWrapper = styled.div`
   margin-top: 24px;
 `;
 
-export function ChooseDestinationContainer() {
-  const [isModalOpen, setModalOpen] = useState(false);
+const AddDestinationButtonWrapper = styled.div`
+  width: 100%;
+  margin-top: 24px;
+`;
+
+const StyledAddDestinationButton = styled(Button)`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 8px;
+  width: 100%;
+`;
 
+export function ChooseDestinationContainer() {
   const router = useRouter();
-  const { configuredSources, configuredDestinations, resetState } = useAppStore((state) => state);
+  const { createSources, loading: sourcesLoading } = useSourceCRUD();
+  const { createDestination, loading: destinationsLoading } = useDestinationCRUD();
+  const { configuredSources, configuredFutureApps, configuredDestinations, resetState } = useAppStore((state) => state);
 
   const isSourcesListEmpty = () => {
```

Comment:
this can be simplified:
const isSourcesListEmpty = () => {
  return !Object.values(configuredSources).some((sources) => sources.length > 0);
};